chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -104,7 +104,7 @@
       for type in clone ms primitive; do
         for resource in $(crm configure show | grep ^$type | cut -d " " -f2);
         do
-          crm resource stop $resource
+          crm --force --wait resource stop $resource
```

Comment:
Is not `--wait` alone enough? This `--force` is for user interaction, is happening with this `--wait` option?